src/components/form/date-input/date-input.html

```diff
@@ -36,4 +36,40 @@
            (blur)="inputBlur()">
   </div>
 
+  <div class="form-group form-group-hour" *ngIf="dateTime">
+    <label class="form-label" [for]="hourId()">Hour</label>
+    <input class="form-control"
+           [id]="hourId()"
+           [name]="hourId()"
+           type="number"
+           [value]="displayHour"
+           (change)="hourChange($event)"
```

Comment:
passing the event to the method is not recommended. Can we access the hour value using a  template reference variable:
on Angular doc there's a better approach:
https://angular.io/guide/user-input#get-user-input-from-a-template-reference-variable